CODE_OF_CONDUCT.md

```diff
@@ -2,13 +2,13 @@
 
 ## 1. Purpose
 
-A primary goal of the UI-Kit is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are
+A primary goal of the components is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are
 committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status,
 and religion (or lack thereof).
 
 This code of conduct outlines our expectations for all those who participate in our community, as well as the consequences for unacceptable behavior.
 
-We invite all those who participate in the UI-Kit to help us create safe and positive experiences for everyone.
+We invite all those who participate in the components to help us create safe and positive experiences for everyone.
```

Comment:
We invite all those who participate and contribute to the design system components to help us create safe and positive experiences for everyone.